build-system-tests/scripts/mega-app-create-app.sh

```diff
@@ -109,7 +109,7 @@ if [[ "$FRAMEWORK" == 'vue' ]]; then
     npm install -g @vue/cli@${BUILD_TOOL_VERSION}
     if [ "$BUILD_TOOL" == 'vue-cli' ]; then
         echo "vue create --preset ../templates/components/vue/preset-${FRAMEWORK_VERSION}.json $MEGA_APP_NAME"
-        vue create --preset ../templates/components/vue/preset-${FRAMEWORK_VERSION}.json $MEGA_APP_NAME
+        printf 'Y\n' | vue create --preset ../templates/components/vue/preset-${FRAMEWORK_VERSION}.json $MEGA_APP_NAME
```

Comment:
I think iirc echo prints the end of line by default.

```suggestion
        echo 'Y' | vue create --preset ../templates/components/vue/preset-${FRAMEWORK_VERSION}.json $MEGA_APP_NAME
```